nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -198,6 +203,12 @@ def _permuted_ols_on_chunk(
         hypothesis is that the effect is zero or negative.
         Default=True
 
+    tfce : :obj:`bool`, optional
```

Comment:
Is there a reason why you didn't add docstrings for the other parameters you added (`masker` and `tfce_original_data`)?

Reply:
Sorry about that. It was an oversight on my part. I've now added the parameter descriptions though.